frontend/libs/studio-components/src/components/StudioPageHeader/StudioPageHeaderPopoverTrigger/StudioPageHeaderPopoverTrigger.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { forwardRef } from 'react';
+import type { StudioPopoverTriggerProps } from '../../StudioPopover';
+import { StudioPopover } from '../../StudioPopover';
+import cn from 'classnames';
+import classes from '../common.module.css';
+
+import type { StudioPageHeaderColor } from '../types/StudioPageHeaderColor';
+import type { StudioPageHeaderVariant } from '../types/StudioPageHeaderVariant';
```

Comment:
I guess this was done automatically by Eslint. I've seen that it does this sometimes when running the `fix` command, but I don't know why, since it doesn't complain when I remove the line break. I'll remove this one too, nice catch!

Reply:
Hm, weird, maybe something we can consider fixing in the eslint config 🤷